zerver/forms.py

```diff
@@ -25,6 +28,15 @@
                     u"<a href=\"https://blogs.dropbox.com/tech/2015/09/open-sourcing-zulip-a-dropbox-hack-week-project/\">Zulip is open source</a>, so you can install your own Zulip server " + \
                     u"by following the instructions on <a href=\"https://www.zulip.org\">www.zulip.org</a>!"
 
+
+def get_registration_string(domain):
+    # type: (unicode) -> unicode
+    register_url  = reverse('register') + domain
+    register_account_string = 'The organization with the domain already exists. '+ \
+                              'Please register your account <a href=' + register_url + '>here</a>.'
+
+    return _(register_account_string.decode("utf-8"))
```

Comment:
you should call _() on the actual strings, otherwise the tooling for extracting strings for translation `./manage.py makemessages` won't extract the strings properly.  See  https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#standard-translation for more information on this.

You want to do something like this example: `_('Today is %(month)s %(day)s.') % {'month': m, 'day': d}`